contracts/ModuleRegistry.sol

```diff
@@ -208,8 +228,10 @@ contract ModuleRegistry is IModuleRegistry, EternalStorage {
         uint256 moduleType = getUintValue(Encoder.getKey("registry", _moduleFactory));
 
         require(moduleType != 0, "Module factory should be registered");
+        // Ordering (lazy evaluation) in below require statement is important. A moduleFactory should not be able
```

Comment:
This code comment isn't valid anymore, is it?

Reply:
removed